acbuild/copy.go

```diff
@@ -20,9 +20,9 @@ import (
 
 var (
 	cmdCopy = &cobra.Command{
-		Use:     "copy PATH_ON_HOST PATH_IN_ACI",
+		Use:     "copy PATH_ON_HOST... PATH_IN_ACI",
```

Comment:
I'm a little torn here; it would be great to be explicit about these different operation modes. Any other ideas?